api/v1alpha1/generated.proto

```diff
@@ -27,6 +27,57 @@ import "sigs.k8s.io/controller-runtime/pkg/scheme/generated.proto";
 // Package-wide variables from generator "generated".
 option go_package = "v1alpha1";
 
+// RouteMatchExtensionObjectReference identifies a route-match extension object
```

Comment:
This seems wrong. It looks like the types.go is correct but the generated proto is not? Any chance this is just out of date and our tests aren't picking up the issue?

Reply:
I've run multiple tests and it appears that `go-to-protobuf` may have a bug. I filed https://github.com/kubernetes/code-generator/issues/107. @howardjohn PTAL when you have a moment and let me know if you have any suggestions.

Reply:
@robscott [the bug](https://github.com/kubernetes-sigs/service-apis/blob/master/api/v1alpha1/generated.proto#L664-L668) you identified existed prior to this PR.